index.js

```diff
@@ -78,6 +83,196 @@ function loader (version) {
         return new Item(item.blockId, item.itemCount, item.itemDamage, item.nbtData)
       }
     }
+
+    static getEnchants (item) {
+      const normalize = (enchList) => enchList.map(ench => {
+        return { lvl: ench.lvl, name: mcData.enchantments[ench.id].name }
+      })
```

Comment:
Why declare this function if its only used once ? In general, avoid declaring functions in other functions, and if they are short like this and only used once, don't make a function.